src/applications/personalization/profiles-wrapper/ProfilesWrapper.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Suspense } from 'react';
+import { connect } from 'react-redux';
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import Profile2Wrapper from 'applications/personalization/profile-2/components/Profile2Wrapper';
+import ProfileOneWrapper from 'applications/personalization/profile360/containers/VAProfileApp';
+import routes from 'applications/personalization/profile-2/routes';
+import {
+  isInMVI as isInMVISelector,
+  isLOA1 as isLOA1Selector,
+  isLOA3 as isLOA3Selector,
+} from 'platform/user/selectors';
+import { selectShowProfile2 } from 'applications/personalization/profile-2/selectors';
+
+const LoadingPage = <Profile2Wrapper>Loading...</Profile2Wrapper>;
+
+const ProfilesWrapper = ({ showProfile1, isLOA1, isLOA3, isInMVI }) => {
+  // On initial render, both isLOA props are false.
+  // We need to make sure the proper redirect is hit,
+  // so we show a loading state till one value is true.
+  if (!isLOA1 && !isLOA3) {
+    return (
+      <BrowserRouter>
+        <Suspense fallback={LoadingPage} />
+      </BrowserRouter>
+    );
+  }
+
+  if (showProfile1) {
+    return <ProfileOneWrapper />;
+  }
+
+  return (
+    <BrowserRouter>
+      <Suspense fallback={LoadingPage}>
+        <Switch>
+          {routes.map(route => {
+            if ((route.requiresLOA3 && !isLOA3) || !isInMVI) {
+              return (
+                <Redirect
+                  from={route.path}
+                  key="/profile/account-security"
+                  to="/profile/account-security"
+                />
+              );
+            }
+
+            const Component = route.component;
+
+            return (
+              <Route
+                component={props => (
+                  <Profile2Wrapper {...props}>
+                    <Component />
+                  </Profile2Wrapper>
+                )}
+                exact
+                key={route.path}
+                path={route.path}
+              />
+            );
+          })}
+
+          <Redirect
+            exact
+            from="/profile"
+            key="/profile/personal-information"
+            to="/profile/personal-information"
+          />
+        </Switch>
+      </Suspense>
+    </BrowserRouter>
+  );
+};
+
+const mapStateToProps = state => {
+  const profileVersion = localStorage.getItem('PROFILE_VERSION');
```

Comment:
I just realized we should use our helper to access `localStorage` like so: `import localStorage from 'platform/utilities/storage/localStorage';` Info as to why we do this at the top of the `localStorage` file https://github.com/department-of-veterans-affairs/vets-website/blob/92811be079cac99884abc28c2c96da61f08fe2a5/src/platform/utilities/storage/localStorage.js#L1-L14

Reply:
Oh and we should keep this `'PROFILE_VERSION'` in a shared constant somewhere as I'm gonna need it in some work I'm doing :)